webui/cypress/integration/server-details.spec.js

```diff
@@ -45,13 +45,36 @@ describe('Server details', () => {
     cy.get('.meta-test__ServerDetailsModal button').contains('Storage').click();
     cy.get('.meta-test__ServerDetailsModal button').contains('Network').click();
     cy.get('.meta-test__ServerDetailsModal button').contains('General').click();
+    cy.get('.meta-test__ServerDetailsModal button').contains('Issues 0').click();
+
     cy.get('.meta-test__ServerDetailsModal').closest('div').find('.meta-test__youAreHereIcon');
+
+    cy.get('.meta-test__ServerDetailsModal button:contains(Select zone)').click();
+    cy.get('div').contains('You have no any zone,');
+    cy.get('div').contains('please add one.')
+    cy.get('button:contains(Add new zone)').click();
+    cy.get('.ZoneAddModal input[name="uri"]').type('Narnia');
+    cy.get('.meta-test__ZoneAddSubmitBtn').click();
+    cy.get('.ZoneAddModal').should('not.exist');
+    cy.get('.meta-test__ServerDetailsModal').find('button:contains(Zone Narnia)');
+    cy.get('button:contains(Close)').click();
+    cy.get('.meta-test__ServerDetailsModal').should('not.exist');
+    
+    cy.get('li').contains('dummy-1').closest('li')
+      .find('.meta-test__ReplicasetServerListItem__dropdownBtn').click();
+    cy.get('.meta-test__ReplicasetServerListItem__dropdown *')
+      .contains('Server details').click();
+    cy.get('.meta-test__ServerDetailsModal').find('button:contains(Zone Narnia)').click({force: true});
+    cy.get('div').contains('You have no any zone,').should('not.exist');
+    cy.get('button:contains(Add new zone)').should('be.enabled');
+
```

Comment:
Let's add two more cases:

1. Open "dummy-2" server details. Check the zone isn't selected. Check Narnia is in dropdown list. Create new zone "Mordor" for dummy 2. Check the label (Zone Mordor)
1. Open "dummy-1" server details. Check zone suggestions include both Narnia and Mordor. Deselect Narnia. Check the label (Select zone). Check that Narnia isn't suggested anymore.